server/config/disaster_dashboard/Earth.textproto

```diff
@@ -127,6 +127,35 @@ metadata {
       }
     }
   }
+  event_type_spec {
+    key: "heat"
+    value {
+      id: "heat"
+      name: "Extreme Heat"
+      event_type_dcids: "HeatTemperatureEvent"
+      color: "#f01212"
+      default_severity_filter: {
+        prop: "differenceTemperature",
+        unit: "Celsius"
+        upper_limit: 25,
```

Comment:
Why do we need to have an upper limit? Is the assumption that this would indicate a bug?

Reply:
I think it's mostly because we are not handling the case where it's unspecified well. I'll add as a TODO for us to handle it.

Reply:
That sounds Ok. Could we increase this limit so that we aren't excluding any amazingly severe events? I suggest an upper_limit of 35? Ditto for the cold events?

Reply:
done